### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aditya28sarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,60 @@ Documentation: [https://neumorphismreact.netlify.app/](https://neumorphismreact.
 npm i neumorphism-react-design
```
Comment (dsnehasish74) [inline README.md:12]: Great work @aditya28sarin  Just one request Make the heading and documentation link and support button center.

**Review by dsnehasish74 — CHANGES_REQUESTED** [review]
Please do this little changes

**Review by aditya28sarin — COMMENTED** [review]
I have made the changes

**Comment by dsnehasish74** [comment]
Great Work @aditya28sarin

**Pull request merged**